README.md

```diff
@@ -31,6 +31,31 @@ The Commerce App is a sample application that demonstrates the use of ListFeed p
 
 [Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Commerce)
 
+### SimpleCalculator App
```

Comment:
```suggestion
### Simple Calc App
```
To match the workshop and name of the sample app

Reply:
Also try to update any subsequent uses of _SimpleCalculator_